examples/00-mapdl-examples/bracket_static.py

```diff
@@ -0,0 +1,743 @@
+# Create line fillet
+# ~~~~~~~~~~~~~~~~~~
+#
+# The right angle between the two boxes will be improved using a fillet with a
+# radius of :math:`0.4`.
+# This can be done by selecting the lines around that area and creating a fillet.
+#
+# The APDL :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command is used
+# to select lines. Here we use the X and Y location of the lines used to create
+# the boxes to create our selection.
+#
+# After we select the line it needs to be written to a parameter so we can use
+# it to generate the fillet line.
+# That is done using the :meth:`mapdl.get() <ansys.mapdl.core.Mapdl.get>` command
+# from PyMAPDL or `GET` in pymapdl.
+#
+# Since we have selected one line, we can use the ``MAX`` and ``NUM`` arguments
+# for the :meth:`mapdl.get() <ansys.mapdl.core.Mapdl.get>` command.
+#
+# Select first line for fillet
+line1 = mapdl.lsel("S", "LOC", "Y", box1[2])
+l1 = mapdl.get("line1", "LINE", 0, "NUM", "MAX")
+
+
+###############################################################################
+#
+# If we write the command to a python parameter (``line1``) we can use either
+# the APDL parameter ``l1`` or the python parameter ``line1`` when we create
+# the fillet line.
+#
+# Select second line for fillet, create python parameter
+line2 = mapdl.lsel("S", "LOC", "X", box2[0])
+l2 = mapdl.get("line2", "LINE", 0, "NUM", "MAX")
+
+###############################################################################
+# Once we have both lines selected we can use the APDL command
+# :meth:`mapdl.lfillt() <ansys.mapdl.core.Mapdl.lfillt>` to generate the fillet
+# between the lines.
+#
+# **Note** that python could return a list if more than one line is selected.
+
+###############################################################################
+# Here we use a mix of the APDL parameter as a string ``line1`` and the ``l2``
+# python parameter to create the fillet line.
+#
+# Create fillet line using selected line (parameter names)
+fillet_radius = 0.4
+mapdl.allsel()
+line3 = mapdl.lfillt("line1", l2, fillet_radius)
+
+mapdl.allsel()
+mapdl.lplot(vtk=True, cpos="xy")
+
+###############################################################################
+# Create fillet area
+# ~~~~~~~~~~~~~~~~~~
+#
+# To create the area delineated by ``line1``, ``line2`` and newly created
+# ``line3``, we use the :meth:`mapdl.al() <ansys.mapdl.core.Mapdl.al>` command.
+# The three lines are the input. If we select them all, we can use the ``'ALL'``
+# argument to create the area.
+#
+# First we have to reselect the newly created lines in the fillet area.
+# To do this we can use the fillet_radius parameter and the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command.
+#
+# For the two newly created straight lines, the length will be the same as the
+# fillet_radius. So we can use the length argument with the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command.
```

Comment:
```suggestion
# For the two newly created straight lines, the length is the same as the
# ``fillet_radius`` value. Thus, you can use the length argument with the
# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command.
```
Sorry if earlier suggested changes form command to method were technically incorrect!